internal/rukpak/source/containers_image.go

```diff
@@ -45,14 +46,9 @@ func (i *ContainersImageRegistry) Unpack(ctx context.Context, bundle *BundleSour
 	// Resolve a canonical reference for the image.
 	//
 	//////////////////////////////////////////////////////
-	imgRef, err := reference.ParseNamed(bundle.Image.Ref)
+	imgRef, canonicalRef, _, err := resolveReferences(ctx, bundle.Image.Ref, i.SourceContext)
```

Comment:
@joelanford I am curious about this change. What triggered this change?

Reply:
We are moving from a containers/image method to local implementation, hence the question.

Reply:
Nevermind I saw that it is just a wrapper on top of ` reference.ParseNamed(ref)`

Reply:
It was based on a suggestion from catalogd's implementation PR for containers/image. I'm trying to keep them as close to the same as I can.